pom.xml

```diff
@@ -39,6 +39,8 @@
     <properties>
         <version.org.jboss.logmanager>2.1.17.Final</version.org.jboss.logmanager>
         <version.org.slf4j>1.7.30</version.org.slf4j>
+        <!-- Test dependencies -->
+        <version.junit>4.12</version.junit>
```

Comment:
Just a minor comment we should probably use 4.13.1 or GitHub will complain about a security issue :)